crypto/x509/name_print.c

```diff
@@ -178,6 +178,10 @@ static int do_name_ex(BIO *out, const X509_NAME *n, int indent,
     } else {
```

Comment:
From the documentation of `OBJ_obj2nid()` called on l. 174, https://github.com/aws/aws-lc/blob/cfce39dccb49bc61252784c5270c928a51ecace6/include/openssl/obj.h#L110-L113, the `else` branch should be for a known nid. If we prefer to be safe still, maybe the check for NULL would be inside the `else` branch.

Reply:
I think it's safer to have the `NULL` check right before calling `strlen` unconditionally?